<!DOCTYPE HTML>
<!--
	Spatial by TEMPLATED
	templated.co @templatedco
	Released for free under the Creative Commons Attribution 3.0 license (templated.co/license)
	modified from https://github.com/tkipf/tkipf.github.io
-->
<html>

<head>
	<title>Chieh Lin (林潔) | PhD Candidate of Machine Learning @ Carnegie Mellon University</title>
	<meta http-equiv="content-type" content="text/html; charset=utf-8" />
	<meta name="description" content="" />
	<meta name="keywords" content="" />
	<noscript>
		<link rel="stylesheet" href="css/style.css" />
		<link rel="stylesheet" href="css/skel.css" />
		<link rel="stylesheet" href="css/style-xlarge.css" />
	</noscript>
	<link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Raleway:400,700">
	<link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Open+Sans">
	<link rel="stylesheet" href="css/font-awesome.min.css">
	<link rel="icon" type="image/png" sizes="32x32" href="/favicon.ico">
	<!--<<link rel="icon" type="image/png" sizes="16x16" href="/favicon.png">-->
	<!--[if lte IE 8]><script src="js/html5shiv.js"></script><![endif]-->
	<script src="js/jquery.min.js"></script>
	<script src="js/skel.min.js"></script>
	<script src="js/skel-layers.min.js"></script>
	<script src="js/init.js"></script>
	<script src='https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML'></script>
</head>

<body class="landing">

	<!-- Header -->
	<header id="header">
		<ul class="icons">
			<li>
				<a href="https://scholar.google.com/citations?user=Ddz61kgAAAAJ&hl=zh-TW" class="icon fa-graduation-cap"></a>
			</li>
			<li>
				<a href="https://github.com/jessica1338" class="icon fa-github"></a>
			</li>
			<li>
				<a href="https://www.linkedin.com/in/chieh-lin-925a327b/" class="icon fa-linkedin"></a>
			</li>
		</ul>
		<nav id="nav">
			<ul>
				<li><a href="/" class="active">Home</a></li>
				<li><a href="/#two">Research</a></li>
				<li><a href="/projects/">Projects</a></li>
				<li><a href="/#footer">Contact</a></li>
			</ul>
		</nav>
	</header>

	<!-- One -->
	<section id="one" class="wrapper style1">
		<div class="container 75%">
			<div class="row 200%">
				<div class="6u 12u$(medium)">
					<header class="major">
						<h2>Chieh Lin (林潔)</h2>
						<p>
							<div class="image rounded" style="margin-bottom: 1.5em;"><img src="images/chiehlin.jpg" width="200" alt="" /></div>
						</p>
						<p>PhD candidate of Machine Learning</p>
						<p>Carnegie Mellon University</p>
					</header>
				</div>
				<div class="6u$ 12u$(medium)">
					<p>I am a fifth-year PhD student in Machine Learning at the Carnegie Mellon University, advised by <a href="http://www.cs.cmu.edu/~zivbj/" target="_blank">Prof. Ziv Bar-Joseph</a> , who leads <a href="http://sb.cs.cmu.edu/" target="_blank">Systems Biology Group</a>. I plan to graduate before May 2020. My main area of interest is applying/designing Machine Learning models for solving real-world problems, including (but not limited to) understanding complex biological systems, drug discovery, healthcare problems, etc. </p>

					<p>When I was a student in National Taiwan University, I joined <a href="https://www.cmdm.tw/index.html" target="_blank"> Computational Molecular Design and Biomarker Detection in Metabolomics lab </a>, lead by <a href="https://www.cmdm.tw/janetseng.html" target="_blank">Prof. Yufeng Jane Tseng</a> 
					<!--had exposure to a number of fields and&mdash;after a short interlude in Neuroscience-related research</p>
						at the <a href="http://brain.mpg.de/research/helmstaedter-department.html" target="_blank">Max Planck Institute for Brain Research</a>&mdash;eventually found my way into Machine Learning.</p>-->
				</div>
			</div>
		</div>

	</section>

	<!-- Two -->
	<section id="two" class="wrapper style2">
		<div class="container 75%">
			<h2 style="text-align: center">Selected Research Projects and Publications</h2>
			<hr />
			<div class="row 200%">
				<div class="4u 6u(large) 12u$(medium)">
						<div class="image"><img src="images/BU_CSHMM.png" width="100%" alt="" style="border:none;" /></div>
				</div>
				<div class="8u$ 6u$(large) 12u$(medium)">
					<header>
						<h4>Single-cell time-series mapping of cell fate trajectories reveals an expanded developmental potential for human PSC-derived distal lung progenitors </h4>
						<p> CSHMM predicts the precise timing of Wnt modulation that maintains lung cell fate.</p>
					</header>
					<p>Killian Hurley, Jun Ding, ... , <span style="text-decoration: underline;">Chieh Lin</span>, ... , Ziv Bar-Joseph, and Darrell N Kotton.  <span style="font-weight: bold">Single-cell time-series mapping of cell fate trajectories reveals an expanded developmental potential for human PSC-derived distal lung progenitors </span>, (under revision in Cell Stem Cell) [<a href='https://www.biorxiv.org/content/biorxiv/early/2019/09/26/782896.full.pdf'>PDF(bioarxiv)</a>]</p>
						<p> In this project, we plan to use CSHMM to improve the differentiation protocol for lung cell types. We showed that CSHMM can not only identify the relevant signalling pathways which determine cell fate, but can also predict with precision the timing of pathway modulation to increase differentiation efficiency to the target cell type.  </p>
				</div>
				
			</div>
			<hr />
			<div class="row 200%">
				<div class="4u 6u(large) 12u$(medium)">
						<div class="image"><img src="images/LINTIMAT.png" width="100%" alt="" style="border:none;" /></div>
				</div>
				<div class="8u$ 6u$(large) 12u$(medium)">
					<header>
						<h4>Single-cell Lineage Tracing by Integrating CRISPR-Cas9 Mutations with Transcriptomic Data </h4>
						<p> LinTIMat is a new model that combines both mutation and expression data for reconstructing better cell lineage tree</p>
					</header>
					<p>Hamim Zafar*, <span style="text-decoration: underline;">Chieh Lin*</span>, Ziv Bar-Joseph, <span style="font-weight: bold">Single-cell Lineage Tracing by Integrating CRISPR-Cas9 Mutations with Transcriptomic Data </span>, (under review in Nature Communications) [<a href='https://github.com/jessica1338/LinTIMaT'>github</a>, <a href='https://www.biorxiv.org/content/biorxiv/early/2019/05/07/630814.full.pdf'>PDF(bioarxiv)</a>, <a href='paper_pdf/LinTIMaT_HZ.pdf'> ISMB/ECCB 2019 Poster</a>] (* equally-contributed, order determined by coin flip)</p>
						<p>We developed a novel method, LinTIMaT, which learns a probabilistic model that reconstructs cell lineages by integrating mutation and expression data. Analysis shows that expression data helps resolve the ambiguities arising in when lineages are inferred based on mutations alone, while also enabling the integration of different individual lineages for the reconstruction of a consensus lineage tree. </p>
				</div>
				
			</div>
			<hr />
			<div class="row 200%">
				<div class="4u 6u(large) 12u$(medium)">
						<div class="image"><img src="images/CSHMMTF.png" width="100%" alt="" style="border:none;" /></div>
				</div>
				<div class="8u$ 6u$(large) 12u$(medium)">
					<header>
						<h4>Inferring TF activation order in time series scRNA-Seq studies</h4>
						<p> We desing a new probabilistic model that utilize transcrition factor-target (TF-target) relationship and is capable to assign continuous activation time of TFs</p>
					</header>
					<p><span style="text-decoration: underline;">Chieh Lin</span>, Jun Ding, Ziv Bar-Joseph, <span style="font-weight: bold">Inferring TF activation order in time series scRNA-Seq studies</span>, (under revision in PLOS Computational Biology) [<a href='https://github.com/jessica1338/CSHMM-TF-for-time-series-scRNA-Seq'>github</a>]</p>
						<p> We developed the Continuous-State Hidden Markov Models TF (CSHMM-TF) method which integrates probabilistic modeling of scRNA-Seq data with the ability to assign TFs to specific activation points in the model. TFs are assumed to influence the emission probabilities for cells assigned to later time points allowing us to identify not just the TFs controlling each path but also their order of activation.  </p>
				</div>
				
			</div>
			<hr />
			<div class="row 200%">
				<div class="4u 6u(large) 12u$(medium)">
						<div class="image"><img src="images/CSHMM_paper_pic.png" width="100%" alt="" style="border:none;" /></div>
				</div>
				<div class="8u$ 6u$(large) 12u$(medium)">
					<header>
						<h4>Continuous State HMMs for Modeling Time Series Single Cell RNA-Seq Data</h4>
						<p> Designing a probabilistic model for learning the continuous process of cell development with time-series single-cell RNA-Seq datasets</p>
					</header>
					<p><span style="text-decoration: underline;">Chieh Lin</span>, Ziv Bar-Joseph, <span style="font-weight: bold">Continuous State HMMs for Modeling Time Series Single Cell RNA-Seq Data</span>, (Bioinformatics 2019) [<a href='https://academic.oup.com/bioinformatics/advance-article/doi/10.1093/bioinformatics/btz296/5481957'>Link</a>, <a
										href="paper_pdf/CSHMM_btz296.pdf">PDF</a>]</p>
						<p> Designing a new method based on continuous-state HMMs (CSHMMs) for representing and modeling time-series scRNA-Seq data. We provide well-defined CSHMM model and efficient learning and inference algorithms which allow the method to determine both the structure of the branching process and the assignment of cells to these branches. We show that the CSHMM method accurately infers branching topology and correctly and continuously assign cells to paths, improving upon prior methods proposed for this task. Analysis of genes based on the continuous cell assignment identifies known and novel markers for different cell types.</p>
				</div>
				
			</div>
			<hr />
			<div class="row 200%">
				<div class="4u 6u(large) 12u$(medium)">
						<div class="image"><img src="images/SNP.jpeg" width="100%" alt="" style="border:none;" /></div>
				</div>
				<div class="8u$ 6u$(large) 12u$(medium)">
					<header>
						<h4>Cell lineage inference from SNP and scRNA-Seq data</h4>
						<p> Using sequence mutations from scRNA-Seq data to improve the model accuracy of reconstructing developmental trajectories</p>
					</header>
					<p>Jun Ding, <span style="text-decoration: underline;">Chieh Lin</span>, Ziv Bar-Joseph, <span style="font-weight: bold">Cell lineage inference from SNP and scRNA-Seq data</span>, (Nucleic Acids Research 2019),  [<a href='https://academic.oup.com/nar/article/47/10/e56/5367412'>Link</a>, <a
										href="paper_pdf/SNP_gkz146.pdf">PDF</a>].</p>
					<p> We develop a new method to detect significant, cell type specific, sequence mutations from scRNA-Seq data. We show that only a few mutations are enough for reconstructing good developmental trajectories. Integrating these mutations with expression data further improves the accuracy of the reconstructed models. As we show, the majority of mutations we identify are likely RNA editing events indicating that such information can be used to distinguish cell types. </p>
					
				</div>
			</div>
			
			<hr />
			<div class="row 200%">
				<div class="4u 6u(large) 12u$(medium)">
						<div class="image"><img src="images/NN_paper_fig.png" width="100%" alt="" style="border:none;" /></div>
				</div>
				<div class="8u$ 6u$(large) 12u$(medium)">
					<header>
						<h4>Using neural networks for reducing the dimensions of single-cell RNA-Seq data</h4>
						<p> Using domain knowledge to design/improve Machine Learning models for learning the embedding of single cells</p>
					</header>
					<p><span style="text-decoration: underline;">Chieh Lin</span>, Siddhartha Jain, Hannah Kim, Ziv Bar-Joseph, <span style="font-weight: bold">Using neural networks for reducing the dimensions of single-cell RNA-Seq data</span>, (Nucleic Acids Research 2017),  [<a href='https://academic.oup.com/nar/article/45/17/e156/4056711'>Link</a>, <a
										href="paper_pdf/NN_gkx681.pdf">PDF</a>].</p>
					<p> Designing new Neural Network architectures for dimensionality reduction (learning embeddings) of scRNA-Seq data by using domain knowledge of protein-protein interaction (PPI) and transcrition factor-target (TF-target) relationships. We showed that this new embedding improves the peroformance of clustering and retrieval task. We also performed functional analysis of the set of highly weighted nodes for each cell type and showed that even though Neural Networks are often described as a ‘black box’ learning method, many of these are functionally related to the cell type they were selected for.</p>
					
				</div>
			</div>
			<hr />
			<div class="row 200%">
				<div class="4u 6u(large) 12u$(medium)">
						<div class="image"><img src="images/P450.png" width="100%" alt="" style="border:none;" /></div>
				</div>
				<div class="8u$ 6u$(large) 12u$(medium)">
					<header>
						<h4>Rule-based Cytochrome P450 Inhibition Prediction Models</h4>
						<p> Learning rule-based C5.0 classifier for Cytochrome P450 Inhibition Prediction </p>
					</header>
					<p>Bo-Han Su, Yi-shu Tu, <span style="text-decoration: underline;">Chieh Lin</span>, Chi-Yu Shao, Olivia A Lin, Yufeng J Tseng, <span style="font-weight: bold">Rule-based Cytochrome P450 Inhibition Prediction Models</span>, (Journal of Chemical Information and Modeling 2015),  [<a href='https://pubs.acs.org/doi/10.1021/acs.jcim.5b00130'>Link</a>, <a
										href="paper_pdf/P450_acs.jcim.5b00130.pdf">PDF</a>].</p>
					<p>Chi-Yu Shao, Bo-Han Su, Yi-Shu Tu, <span style="text-decoration: underline;">Chieh Lin</span>, Olivia A Lin, Yufeng J Tseng, <span style="font-weight: bold">CypRules: A rule-based P450 inhibition prediction server</span>, (Bioinformatics 2015),  [<a href='https://academic.oup.com/bioinformatics/article/31/11/1869/2365231'>Link</a>, <a
										href="paper_pdf/P450_btv043.pdf">PDF</a>].</p>
					<p>We utilized a rule-based C5.0 algorithm with different descriptors, including PaDEL, Mold2, and PubChem fingerprints, to construct rule-based inhibition prediction models for five major CYP enzymes—CYP1A2, CYP2C9, CYP2C19, CYP2D6 and CYP3A4—that account for 90% of drug oxidation or hydrolysis. We also developed a rational sampling algorithm for the selection of compounds in the training data set, to enhance the performance of these CYP prediction models. Our models significantly outperformed all of the currently available models. Also, they can be used for rapid virtual screening of a large set of compounds due to their ruleset-based nature. Moreover, such rule-based prediction models can provide rulesets for structural features related to the five major CYP enzymes.   </p>
					
				</div>
			</div>
			<hr />
			<div class="row 200%">
				<div class="4u 6u(large) 12u$(medium)">
						<div class="image"><img src="images/PN2S.png" width="100%" alt="" style="border:none;" /></div>
				</div>
				<div class="8u$ 6u$(large) 12u$(medium)">
					<header>
						<h4>Polymer Name to Structure Project</h4>
						<p> We developed a brand new framework that can map various polymer names of the same polymer to its minimal repeating units. This process incorporates data mining, text mining, and computer-aided material design. System can accurately predict simple polymer structures with > 0.9 accuracy </p>
					</header>
					<p>Advances in polymer science have made polymers essential in our everyday life along with unprecedented quantities of data from past several decades. However, to organize such “big data” scattered and accumulated in the text throughout the mass journals, patents, and web pages has been challenging and inefficient due to the complexity and the ambiguity of the polymer representations. We developed a first of its kind automated framework converting various polymer representations to the corresponding polymer structures, PolyName2Structure. Machine learning models and algorithms for handling polymer structures were built to predict polymerization pathway, identify the reacting group(s), and generate repeating units after polymerization. This PolyName2Structure achieved >90% accuracy in prediction of the structure of the polymer listed in a commercial catalog and embodied the first step toward resolving the complexity of the data structure for polymers through building a practical vehicle that enables text mining of structural polymers information. Manuscript in preparation</p>
					
				</div>
			</div>
			<hr />
			<div class="row 200%">
				<div class="4u 6u(large) 12u$(medium)">
						<div class="image"><img src="images/CADD.png" width="100%" alt="" style="border:none;" /></div>
				</div>
				<div class="8u$ 6u$(large) 12u$(medium)">
					<header>
						<h4>Computer-aided drug design projects</h4>
						<p> We proposed candidate molecules that could be new drugs for cancer and schizophrenia drugs by performing computational techniques including fragment docking, virtual screening, LeadOp, LeadOp+R. Target proteins are Topoisomerase II and D-amino acide oxidase (DAO) proteins. The top-ranked molecules have been confirmed more effective by wet-lab experiments. </p>
					</header>
					<p>We collected starting molecules that are known to be the inhibitors of target proteins (Topoisomerase II and D-amino acide oxidase (DAO)), then we identify the important structures as scaffolds. Then, we perform fragment docking to generate a list of fragment that could improve the molecule. With scaffold and possible fragments, we attach the fragments to scaffold with LeadOp or LeadOp+R methods and generates millions of drug candidates. Then we use virtual screening and other criteria such as Lipinski's rule of five to filter the drug candidates and output a final rank list of drug candidates. Our collaborators verify the efficacy of the top-ranked drug candidates and selects some to go through clinical trial process.</p>
					
				</div>
			</div>
			<p style="text-align:center;">For a full list of publications, have a look at my <a href="https://scholar.google.com/citations?user=Ddz61kgAAAAJ&hl=zh-TW">Google Scholar</a> page.</p>
		</div>
	</section>


	<!-- Three -->
	<section id="three" class="wrapper style3 special">
		<div class="container 50%">

			<h2>Curriculum vitae</h2>
			<ul class="actions">
				<li><a href="CV_ChiehLin_20191015.pdf" class="button alt big">Download CV</a></li>
			</ul>
		</div>
	</section>


	<!-- Footer -->
	<footer id="footer">
		<div class="container">
			<h2>Get in touch</h2>
			<p><span style="font-weight: bold">chiehl1[AT]cs[DOT]cmu[DOT]edu</span><br /> Machine Learning Department, Carnegie Mellon University<br />5000 Forbes Avenue, Gates Hillman Center, 8th Floor, Pittsburgh, PA 15213</p>
			<p></p>
			<ul class="icons">
				<li>
					<a href="https://scholar.google.com/citations?user=Ddz61kgAAAAJ&hl=zh-TW" class="icon fa-graduation-cap"></a>
				</li>
				<li>
					<a href="https://github.com/jessica1338" class="icon fa-github"></a>
				</li>

				<li>
					<a href="https://www.linkedin.com/in/chieh-lin-925a327b/" class="icon fa-linkedin"></a>
				</li>
			</ul>
			<ul class="copyright">
				<li>&copy; 2019 Chieh Lin</li>
				<li>Design: <a href="http://templated.co">TEMPLATED</a></li>
			</ul>
		</div>
	</footer>

</body>

</html>
